### DISCUSSION TITLE
Using lastest lib of ReactiveUI in a VSIX package, however still trying to locate Version 7.2.0.0. Any ideas?

### DISCUSSION THREAD

**Discussion by meadisu27** [reactiveui/ReactiveUI#3205] [Q&A]
`System.TypeLoadException: Could not load type 'ReactiveUI.IActivatableView' from assembly 'ReactiveUI, Version=7.2.0.0, Culture=neutral, PublicKeyToken=null'.`

I've added binding redirects... no luck.

Any ideas? Thoughts?

**Comment by glennawatson** [comment]
Vsix need to explicitly include the DLLs that are dependencies. So likely that

**Reply by meadisu27** [reply]
Yep. Verified they are included and packaged up with the VSIX.

**Comment by meadisu27** [comment]
I am guessing ReactiveUI.IActivatableView didn't exist in version 7.2.0.0. The VSIX in question has never had a reference to ReactiveUI older than 9.X. So I am very confused on what is trying to locate that old version of ReactiveUI.

**Comment by glennawatson** [comment]
What other visual studio extensions do you have installed. There isn't necessarily DLL isolation between plugins

**Reply by meadisu27** [reply]
That was one of the first things I checked as well. Unfortunately, it's not my machine, but I did ask for the Help -> About Microsoft Visual Studio -> Copy Info to look for any extensions that appeared as outliers. Unfortunately, there was nothing installed outside of the workloads provided by Microsoft.

**Reply by glennawatson** [reply]
I know the GitHub plugin uses ReactiveUI and quite an old version from memory.

**Reply by meadisu27** [reply]
Yes I found that out as well doing some googling and tried the oldest VSIX they had available (as well as some random ones throughout) in order to see if I could replicate the issue on another machine. No luck with that as well.
